openqa-trigger/schedule-obs.rb

```diff
@@ -45,7 +45,7 @@ def trigger_run(version:)
 
   frequency_minutes_ago = DateTime.now.new_offset(0) - (FREQUENCY/1440.0)
   unless qcow2_image['mtime'] >= frequency_minutes_ago
-    @logger.info("No new build found for #{version}...")
+    @logger.info("No new build found for #{version} right now (#{DateTime.now.new_offset(0)}). Last build: #{qcow2_image['mtime']}")
```

Comment:
`DateTime.now.new_offset(0)` is called twice. It could be assigned to a variable. Also in `mail_notification.rb`. I guess it doesn't make a big difference.